sass/layout/_sidebar-left.scss

```diff
@@ -68,8 +68,24 @@
         height: calc(100% - #{$announcement-bar-height});
     }
 
-    body.announcement-banner-tall--fixed & {
-        height: calc(100% - 40px)
+    .badge {
+        display: inline-block;
+        flex-shrink: 0;
+        margin-left: 6px;
+        margin-right: 3px;
+        padding: 0 6px;
+        min-width: 22px;
+        height: auto;
+        font-size: 12px;
+        font-weight: 600;
+        line-height: 18px;
+        letter-spacing: 0px;
+        text-align: center;
+        color: var(--mention-color);
+        border-radius: 9px;
+        background: var(--mention-bg);
+        -webkit-font-smoothing: subpixel-antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
Is this change intended?

Reply:
the original intention was just to remove the code related to the --tall-- banner. I am going to take a look!

Reply:
This commit was merged to master 4 days ago, https://github.com/mattermost/mattermost-webapp/pull/7357/files#diff-90fded696d50eef5af63163b66aaf08e3ee8190744429e674d8bad25d95a1e7fR74 it removed those lines from master. There should have been some merge conflict  that was not applied correctly and added these lines back as part of my fix. Great catch!

Reply:
Great, thanks!